packages/clay-modal/stories/index.tsx

```diff
@@ -46,6 +46,30 @@ const ModalWithState: React.FunctionComponent<IProps> = ({
 	);
 };
 
+const MyApp: React.FunctionComponent<any> = () => {
+	const [, disptach] = useContext(Context);
```

Comment:
Spelling here. should be `dispatch`

sidenote: I didn't realize you could do `[, dispatch]`. very cool.